1.9/overview/architecture/distributed-process-management.md

```diff
@@ -0,0 +1,83 @@
+---
+post_title: Distributed Process Management
+nav_title: Distributed Process Management
+menu_order: 5
+---
+
+This section describes the management of processes in a DC/OS cluster, from the resource allocation to the execution of a process.
+
+At a high level, this interaction takes place between the DC/OS components when a user launches a process. Communication occurs between the different layers, such as the user interacting with the scheduler, and within a layer, for example, a master communicating with agents.
```

Comment:
Suggest: `...different layers, such as the user interacting with the scheduler, and within a layer, such as a master communicating with agents.`